src/components/subcomponents/Button.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <button class="btn"
+    :style="{background: color}"
+    @click="loadPost()"
+    >
+    {{text}}
+    </button>
+</template>
+<script>
+
+
+export default {
+    name: 'Button',
+    props:{
+        text:{
+            type:String,
+            default:'Tlačítko'
+        },
+        color:{
+            type:String,            
+        },
+       
+    },
+    methods:{
+        loadPost(){
+            console.log('loading...')
```

Comment:
Remove `console.log`. Then, `loadPost` will be empty, so remove it as well.

Reply:
Removed also with methods.